sources/slack/__init__.py

```diff
@@ -161,4 +234,36 @@ def logs_resource() -> Iterable[TDataItem]:
         ):
             yield page_data
 
-    yield logs_resource
+    yield from (channels_resource, users_resource, logs_resource)
+
+    if table_per_channel:
```

Comment:
As for p.2 I'm not sure. 
On the one hand in the current code, the channels will be fetched for each source init. On the other hand, if we move the fetching channels properties to resources, we would need to do this separately for messages and for replies. 
So I guess it depends on the use case. Since the current approach is what was done before, I was conservative with implementation. But if you have a strong opinion on this, I can move channels fetching to resources